assets/css/abstracts/_mixins.scss

```diff
@@ -108,3 +108,13 @@
 		background: transparent;
 	}
 }
+
+// Makes sure long words are broken if they overflow the container.
+@mixin wrapBreakWord {
```

Comment:
Kudos to @haszari for this mixin: https://github.com/woocommerce/storefront/pull/1326.